lib/rules/no-invalid-aria-attributes.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const Rule = require('./_base');
+
+const VALID_ARIA_ATTRIBUTES = new Set([
+  'aria-activedescendant',
+  'aria-atomic',
+  'aria-autocomplete',
+  'aria-busy',
+  'aria-checked',
+  'aria-colcount',
+  'aria-colindex',
+  'aria-colspan',
+  'aria-controls',
+  'aria-current',
+  'aria-describedby',
+  'aria-details',
+  'aria-disabled',
+  'aria-dragged',
+  'aria-dropeffect',
+  'aria-errormessage',
+  'aria-expanded',
+  'aria-flowto',
+  'aria-haspopup',
+  'aria-hidden',
+  'aria-invalid',
+  'aria-label',
+  'aria-labelledby',
+  'aria-level',
+  'aria-live',
+  'aria-modal',
+  'aria-multiline',
+  'aria-multiselectable',
+  'aria-orientation',
+  'aria-owns',
+  'aria-placeholder',
+  'aria-posinset',
+  'aria-pressed',
+  'aria-readonly',
+  'aria-relevant',
+  'aria-required',
+  'aria-rowcount',
+  'aria-rowindex',
+  'aria-rowspan',
+  'aria-selected',
+  'aria-setsize',
+  'aria-sort',
+  'aria-valuemax',
+  'aria-valuemin',
+  'aria-valuenow',
+  'aria-valuetext',
+]);
+
+function getAriaAttributes(node) {
```

Comment:
It seems slightly inefficient to create a list of all these attributes. Might I suggest we just iterate over the list of attributes like this:

```
for (const attribute of attributes) {
  if (attribute.name.startsWith('aria-') && !VALID_ARIA_ATTRIBUTES.has(attribute.name)) {
    // ...
  }
}
``` 

This way, we avoid creating a list. Note: if there's a good reason to create a separate list in the future, we can do it, but it just doesn't seem necessary right now.